src/legacy/core_plugins/metrics/public/components/_vis_with_splits.scss

```diff
@@ -1,11 +1,17 @@
 .tvbSplitVis {
   display: flex;
   flex: 1 0 auto;
-}
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: center;
+  width: 100%;
 
-.tvbSplitVis__split {
-  display: flex;
-  flex: 1 0 0;
-  flex-direction: column;
-  overflow: hidden;
+  &__split {
+    display: flex;
+    flex: 0 1 auto;
```

Comment:
Are you sure you want to restrict allowing the metric to grow? It means it will never get bigger than 200x200, which is the opposite of what a non-split chart will do:

<img width="1159" alt="Screen Shot 2019-03-14 at 12 33 43 PM" src="https://user-images.githubusercontent.com/549577/54374396-6c9e7800-4655-11e9-8ad2-291619e2b569.png">

vs

<img width="1154" alt="Screen Shot 2019-03-14 at 12 33 03 PM" src="https://user-images.githubusercontent.com/549577/54374410-7031ff00-4655-11e9-8adf-41a46aca91b5.png">

Reply:
@cchaos  you are right, my mistake. I think I can revert` 'flex: 1 0 auto;'` but what do you think about next case?: 
![image](https://user-images.githubusercontent.com/20072247/54376283-ca51b500-4693-11e9-9731-ccec22910b0d.png)

The last line bigger than previous one